src/Cameras/targetCamera.ts

```diff
@@ -424,23 +424,24 @@ export class TargetCamera extends Camera {
     }
 
     protected _computeViewMatrix(position: Vector3, target: Vector3, up: Vector3): void {
-        if (this.parent) {
-            const parentWorldMatrix = this.parent.getWorldMatrix();
-            Vector3.TransformCoordinatesToRef(position, parentWorldMatrix, this._globalPosition);
-            Vector3.TransformCoordinatesToRef(target, parentWorldMatrix, this._globalCurrentTarget);
-            Vector3.TransformNormalToRef(up, parentWorldMatrix, this._globalCurrentUpVector);
-            this._markSyncedWithParent();
-        } else {
-            this._globalPosition.copyFrom(position);
-            this._globalCurrentTarget.copyFrom(target);
-            this._globalCurrentUpVector.copyFrom(up);
-        }
+        this._globalPosition.copyFrom(position);

```

Comment:
this values will be false then if there is a parent as we do not take parent into account

Reply:
I don't understand.

The `_globalXXX` variables are first filled with values in local space (position, target, up).

Then, `_globalPosition` is updated with the world position when there's a parent (when no parent, the local position is the world position).

`_globalCurrentTarget` and `_globalCurrentUpVector` variables are only used in this function for the initial `LookAtLHToRef` computation, so there's no need to update them to world values. Maybe I should remove them altogether and directly use `target` and `up` in the `LookAtLHToRef` call?

Reply:
Yeah I fill like we can safely remove them